user-frontend/src/config.js.example

```diff
@@ -0,0 +1 @@
+export const AUTH_SERVICE = 
```

Comment:
value missing

Reply:
This is config example to ensure, that online there is no our config. When somebody clones this repo, they should add missing value, which is auth server address and then rename the file from config.js.example to config.js